airflow/www/static/js/tree/details/content/taskInstance/taskActions/Run.jsx

```diff
@@ -52,7 +58,7 @@ const Run = ({
 
   return (
     <Flex justifyContent="space-between" width="100%">
-      <ButtonGroup isAttached variant="outline">
+      <ButtonGroup isAttached variant="outline" disabled={!canRun}>
```

Comment:
Is this visually different when disabled? I can't really tell from the the screenshot.